lib/ansible/modules/network/junos/junos_config.py

```diff
@@ -314,7 +314,7 @@ def main():
         else:
             module.fail_json(msg='unable to retrieve device configuration')
 
-        result['__backup__'] = str(match.text).strip()
+        result['__backup__'] = match.text.strip()
```

Comment:
Do we need to handle similar fix for junos_template as well?
Also, require separate handling for load of backup config with unicode string.